visualization/app/codeCharta/util/usageDataTracker.ts

```diff
@@ -218,7 +217,6 @@ export function trackEventUsageData(actionType: string, state: State, payload?:
 			!isActionOfType(actionType, HeightMetricActions) &&
 			!isActionOfType(actionType, ColorMetricActions) &&
 			!isActionOfType(actionType, ColorRangeActions) &&
-			!isActionOfType(actionType, InvertColorRangeActions) &&
```

Comment:
@ce-bo FYI: InvertColorRangeAction was only called in tests anymore, as the mapColors are customizable now. Inverting is just switching mapColors.positive and mapColors.negative (also see #1648)

Reply:
This sounds good as long as buttons etc. still keep their behavior as before (also with custom configs).